library/src/main/java/com/mindera/skeletoid/generic/ReachabilityReceiver.java

```diff
@@ -0,0 +1,99 @@
+package com.mindera.skeletoid.generic;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.net.ConnectivityManager;
+import android.net.NetworkInfo;
+import android.os.AsyncTask;
+
+import com.mindera.skeletoid.logs.LOG;
+
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+public class ReachabilityReceiver extends BroadcastReceiver {
```

Comment:
@neteinstein What do you think about adding some javadoc explaining how to use this class?
Just how can we setup this on the manifest or by code